testing-services/src/main/java/io/stargate/testing/metrics/TagMeHttpMetricsTagProvider.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.stargate.testing.metrics;
+
+import io.micrometer.core.instrument.Tags;
+import io.stargate.core.metrics.api.HttpMetricsTagProvider;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Simple {@link HttpMetricsTagProvider} for testing that extracts {@value TAG_ME_HEADER} header and
+ * converts to tag.
+ */
+public class TagMeHttpMetricsTagProvider implements HttpMetricsTagProvider {
```

Comment:
do we really need this special test implementation instead of implementing the production code and using it in tests? We can use the proper header (that carries `tenant_id`) name instead of mock (`host` header). wdyt?
Right now this tenant id extraction is not implemented (and not tested)

Reply:
This is because from the OSS perspective there are no extra tags.. And this tests confirms that anything can be plug-ed in. I am creating a test impl of the `HttpMetricsTagProvider` and registering it, so that I can confirm that the metrics will include the extra tag(s) returned, what ever they are. The tests you are mentioning are for the `c2` repo..